src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx

```diff
@@ -29,51 +28,43 @@ import Sidebar from 'components/sharedComponents/sidebar/Sidebar';
 import ExternalLink from 'components/sharedComponents/ExternalLink';
 import ShareIcon from 'components/sharedComponents/stickyHeader/ShareIcon';
 
-let sections = [
+const cookie = 'usaspending_covid_homepage';
```

Comment:
Can this var be removed?

Reply:
probably? I just accepted all current changes on this page (on master branch, incoming from QAT)